docs/7.x/requirements.md

```diff
@@ -134,10 +134,10 @@ These ports are used for Cluster operation and should be open between cluster no
 | 3012                    | TCP                | HTTPS                | all         | all         | Gravity RPC agent                        |
 
 !!! note "Source/Destination Legend"
-  * all - Any node which is a member of the cluster
-  * ext - Any source outside the cluster
-  * localhost - The port is only used within the host where the request started
-  * controllers - Nodes which are designated "controller" (aka "master") role
+    * all - Any node which is a member of the cluster
+    * ext - Any source outside the cluster
+    * localhost - The port is only used within the host where the request started
+    * controllers - Nodes which are designated "controller" (aka "master") role
```

Comment:
same nit as in https://github.com/gravitational/gravity/pull/1876

Reply:
> nit: Maybe use control plane instead of controller when describing a controller node?

We don't use control plane consistently elsewhere in the docs, so I'm not sure using it here will be any clearer than the admittedly muddy `controllers`.

```
walt@work:~/git/gravity$ rg 'control plane' docs/7.x
docs/7.x/hub.md
3:The Gravity Hub is a multi-cluster control plane available in the Enterprise

docs/7.x/cluster.md
732:Clusters with complex software deployed on them require separation of workloads between the control plane and the application components to enable a seamless upgrade experience.
1428:### Dedicated control plane nodes
```

`master` seems to be the most commonly used terminology.  Does that work for you?

Reply:
Aside from this, the intent touching this line is fixing a blatantly broken indentation level:

![image](https://user-images.githubusercontent.com/187314/87588124-f8844200-c697-11ea-9a86-d6101dfac1eb.png)

I'm not sure I want to take on this scope creep, as changing this will require editing all of the table above.

Reply:
I've decided not to pursue the recommended change in this PR.  Too much scope creep.